boilerplate/openshift/custom-catalog-osd-operator/install-opm.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# global envs
+OPM_VERSION="v1.15.2"
+GOOS=$(go env GOOS)
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+# install opm binary if it does not exist
+if ! which opm >/dev/null 2>&1; then
+  echo "installing opm-binary as it is not installed already"
+  mkdir -p .opm/bin
+  cd .opm/bin
+  opm="opm-${OPM_VERSION}-$GOOS-amd64"
+  opm_download_url="https://github.com/operator-framework/operator-registry/releases/download/${OPM_VERSION}/${GOOS}-amd64-opm"
+  curl -sfL "${opm_download_url}" -o "$opm"
```

Comment:
If this fails, you'll carry on down the road. As written, I think you'd end up actually failing up on [L31 of custom-catalog-build-push.sh](https://github.com/openshift/boilerplate/pull/190/files#diff-2e248d62bbcf4b9ef38e0a16052835a5ac5a55ee57645563c252ac6af78351dbR31), but it could be tougher than necessary to debug because the failure from this curl would be (possibly a lot) earlier in the output.

Consider adding error checking, or using `set -e` in this script.

Reply:
will add `set -e` thanks